app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit distutils-r1 vcs-snapshot
+
+GIT_REF=5abecd53e1d84f2a5fd3fc60d2f5d71e518826c5
+
+DESCRIPTION="Python scripts to manipulate trash cans via the command line"
+HOMEPAGE="https://github.com/andreafrancia/trash-cli"
+SRC_URI="https://github.com/andreafrancia/${PN}/archive/${GIT_REF}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		dev-python/nose[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+	)"
+
+python_test() {
+	nosetests -v || die
+}
```

Comment:
Looks to me like `distutils_enable_tests nose` should work okay here. You do need to add mock dependency separately though.

Reply:
Neat.